pages/redeemed.js

```diff
@@ -191,6 +191,8 @@ class RedeemedPage extends React.Component {
     const { amount, collective, currency, expiryDate, loading } = this.state;
     const error = this.getError();
     const emitter = this.state.emitter || (data && data.Collective);
+    const recommendedCollectives =
+      get(emitter, 'settings.recommendedCollectives') || get(data.Collective, 'settings.recommendedCollectives');
```

Comment:
The second part doesn't look useful? `|| get(data.Collective, 'settings.recommendedCollectives');`

Reply:
Yes. 
Sorry, I didn't do a proper clean up.